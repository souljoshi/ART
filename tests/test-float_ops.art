int main()
{
	
	double a,b,c;
	a=5.5;
	b=6.6;
	c=a+b;
	printf(c);
	c=a-b;
	prints("\n");
	printf(c);
	c=a*b;
	prints("\n");
	printf(c);
	c=a/b;
	prints("\n");
	printf(c);
}