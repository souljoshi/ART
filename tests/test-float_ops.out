12.100000
-1.100000
36.300000
0.833333